app/email_batching.go

```diff
@@ -71,7 +71,7 @@ func NewEmailBatchingJob(s *Server, bufferSize int) *EmailBatchingJob {
 }
 
 func (job *EmailBatchingJob) Start() {
-	mlog.Debug(fmt.Sprintf("Email batching job starting. Checking for pending emails every %v seconds.", *job.server.Config().EmailSettings.EmailBatchingInterval))
+	mlog.Debug("Email batching job starting. Checking for pending emails every", mlog.Int("seconds", *job.server.Config().EmailSettings.EmailBatchingInterval))
```

Comment:
I don't think we need structured logging in this case. It would make sense to keep Sprintf here.

Reply:
done